api/hostmanagement/openamt/openamt.go

```diff
@@ -32,7 +36,7 @@ func NewService(dataStore portainer.DataStore) *Service {
 	}
 	return &Service{
 		httpsClient: &http.Client{
-			Timeout: time.Second * time.Duration(5),
+			Timeout: time.Second * time.Duration(60),
```

Comment:
Should this be configurable?  or at least brought into a constant somewhere.  60s is probably ok, but we're assuming that this value is reasonable.

Reply:
moved it to a const and changed it to 30, I set a very high value because particularly the devices endpoints seem to take longer at least the first time they are invoked, but 30s should be more than enough